apps/rush-lib/src/data/RushHooks.ts

```diff
@@ -0,0 +1,20 @@
+import { IRushHooksJson } from './RushConfiguration';
+import Utilities from '../utilities/Utilities';
+
+export default class RushHooks {
+  private _postCommandHooks: string[];
+
+  public constructor(rushHooksJson: IRushHooksJson) {
+    this._postCommandHooks = rushHooksJson.postCommand;
+  }
+
+  public get postCommandHooks(): string[] {
+    return this._postCommandHooks;
+  }
+
+  public onPostCommand(): void {
+    this._postCommandHooks.forEach((script) => {
+      Utilities.executeShellCommand(script);
+    });
```

Comment:
FYI I've been trying to avoid putting business logic in the configuration class.

RushConfiguration's job is to read (and sometimes write) config files for Rush.